### PR TITLE
word-count: Add test for multiple whitespace

### PR DESCRIPTION
**Pull request by jonmcalder**
See #1022 for reference

### CODE REVIEW

File: exercises/word-count/canonical-data.json [hunk 1]
```diff
@@ -111,6 +111,15 @@
         "large": 2,
         "and": 1
       }
+    },
+    {
+      "description": "with multiple whitespace",
```
Comment (Insti) [inline exercises/word-count/canonical-data.json]: A more helpful description would be nice.
What is it about multiple whitespace that makes it interesting to test?

"space surrounded by spaces not detected as a word"

Comment (jonmcalder) [reply exercises/word-count/canonical-data.json]: Thanks - a valid point. I fell into the trap of just imitating the overly brief descriptions from some of the prior test cases...

**Comment by jonmcalder** [comment]
Thanks @ErikSchierboom!
